packages/web/app/src/components/v2/tooltip.tsx

```diff
@@ -13,13 +13,7 @@ function Wrapper({
 }): ReactElement {
   return (
     <T.Root>
-      <T.Trigger
-        onClick={e => {
-          e.preventDefault();
-        }}
-      >
-        {children}
-      </T.Trigger>
+      <T.Trigger asChild>{children}</T.Trigger>
```

Comment:
to avoid warning while adding a tooltip for `<button />`
![image](https://user-images.githubusercontent.com/7361780/224867836-bb8f2a69-7513-4de5-96e3-68d1971c0356.png)